src/SprykerEco/Zed/Easycredit/Business/EasycreditFacadeInterface.php

```diff
@@ -23,6 +23,11 @@ interface EasycreditFacadeInterface
     /**
      * Specification:
      * - Send a request to Easycredit for payment initialization.
+     * - Requires `QuoteTransfer.totals` parameter.
```

Comment:
```suggestion
     * - Requires `QuoteTransfer.totals` parameter to be set.
```

Same for other cases.